lessons/beginners/def/index.md

```diff
@@ -167,9 +264,11 @@ y = float(input('Zadej délku poloosy 2: '))
 print('Obsah je', obsah_elipsy(x, y))
 ```
 
-Samozřejmě existují výjimky: funkce která přímo vytváří textový výpis,
-může používat `print`; funkce která načítá textové informace zase `input`.
-Když ale funkce něco počítá, je dobré v ní `print` ani `input` nemít.
+Samozřejmě existují výjimky: procedura která přímo vytváří textový výpis
+(např. tabulku) může používat `print`; funkce která načítá textové informace
+(jako `ano_nebo_ne` výše) zase `input`.
+Když ale funkce něco *počítá*, nebo když si nejsi jist{{gnd('ý', 'á')}},
+je dobré ve funkci `print` ani `input` nemít.
```

Comment:
GitHubu umí i [víceřádkové návrhy](https://github.blog/changelog/2020-02-26-multi-line-code-suggestions-beta/). Jen nevím jestli je to stále beta, nebo už to je pro všechny uživatele.
```suggestion
Samozřejmě existují výjimky: procedura, která přímo vytváří textový výpis
(např. tabulku), může používat `print`; funkce, která načítá textové informace
(jako `ano_nebo_ne` výše), zase `input`.
Když ale funkce něco *počítá*, nebo když si nejsi jist{{gnd('ý', 'á')}},
je dobré ve funkci `print` ani `input` nemít.
```